tour-tip/config.js

```diff
@@ -0,0 +1,51 @@
+export default [
```

Comment:
Changed this from a `.json` to a `.js` because I was having a nightmare using it in tests (even though really I should be mocking it out. More on this below.)